storage/remote/dial_context_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
Not that it really matters legally, but we usually put the current year here when creating new files.